environment_dasmon.yml

```diff
@@ -6,6 +6,6 @@ dependencies:
 - python=3.7
 - django=1.11.25
 - stomp.py=7.0.0
-- psycopg2=2.8.6
 - setuptools
 - postgresql
+- psycopg2-binary=2.8.6
```

Comment:
webref uses the binary package, mostly for consistency across projects.  The main difference I'm aware of is `binary` is precompiled.  The other is from source and requires compilation, unless we need to compile it ourselves for some reason I imagine binary is marginally better for build times, otherwise negligible.

Reply:
So far I've found [this article](https://pysql.tecladocode.com/section05/lectures/04_psycopg2_vs_psycopg2-binary/) saying that `-binary` is preferred, but [psycopg suggests](https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary) that you should build from source. Since conda has a standard way of build/deploy with consistent tooling, it is probably fine to go with `-binary`.